### ISSUE TITLE
Dear Bambu, Pause without artifacts. 

### ISSUE THREAD

**Issue by Gizmo6** [bambulab/BambuStudio#4248]
Dear Bambu Lab,

I have a Firmware change suggestion.

When your machines have to pause due to a number of things that may go wrong. After a set time it de-energises the stepper motors.

This is undesirable because it loses its physical position. Even when it rehomes it can be ever so slightly misaligned. 

I would rather the machines kept the steppers energised no matter how long the wait is to prevent these misalignement artifacts on resume.

This situation rarely occurs for machines with ams with the auto swap feature enabled as the machine stays on while it swaps over to the next spool.

If however the ams is unable to swap it will, after some time de-engerise and lose its position ever so slightly. You can hear when this occurs as there is often a clunk sound.

So please take this into consideration in the next firmware update.

(even rehoming isn't 100% fool proof due to ever so slight sensor detection differences.)

Its more a small quality of life edit which can make better resumed prints.

**Comment by darkneon2002** [comment]
Yep, it's terrible. Every time I program a pause, you can see where it was. Compared to sitting there and waiting for the right moment to manually pause the printer, it will be invisible. But who has the time for that, especially if it needs to pause multiple times?
